.github/workflows/builds.yml

```
@@ -9,11 +9,12 @@ on:
 
 env:
   CTEST_OUTPUT_ON_FAILURE: 1
+  ACTS_CI_IMAGE_VERSION: v1
```

I would suggest to use
```suggestion
  ACTS_CI_IMAGE_TAG: v1
```
to be consistent with the Docker nomenclature.